packages/relay-runtime/util/__tests__/resolvers/__generated__/RelayReaderResolverTest1FragmentRefetchableQuery.graphql.js

```diff
@@ -0,0 +1,142 @@
+/**
```

Comment:
Cool. This only generates a normalization artifact, so there's not reference to the actual resolver module. 👍